src/components/DetailModal.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+
+import arrowLeft from '../image/arrow-left.svg';
+import '../CSS/Icon.css';
+
+const DetailModal = ({ show, handleClose, item }) => {
+  let lastPurchase =
+    item && item.last_purchased !== ''
+      ? new Date(item.last_purchased).toDateString()
+      : '';
+
+  return (
+    <Modal show={show} onHide={handleClose}>
+      <Modal.Header>
+        <Button onClick={handleClose} className="to-shopping">
+          <img
+            className="arrow-left"
+            src={arrowLeft}
+            alt="Back to shopping list"
+          />{' '}
+          <span className="back-text">Back to list</span>
+        </Button>
+        <Modal.Title>
+          <h3>{item.name} details</h3>
```

Comment:
Should this be a Heading 1 instead of a Heading 3?

It's the page's title so an H1 is fitting, and for accessibility you need to keep the headings in order. This is currently H3, H2, H2. It might be better as H1, paragraph, paragraph. The H2's are not followed by body copy so paragraph tags or a list would be better for page structure and accessibility.